choreographer/src/main/resources/application.properties

```diff
@@ -35,13 +35,10 @@ log_all_request_and_response=false
 sr_address=127.0.0.1
 sr_port=8443
 
-# If true, using notify URL address with self-addressing IPv4, IPv6 and no-type addresses are allowed.
-# In case of self-addressing addresses the IP packets cannot be directed from one device to another.
-allow_self_addressing=true
-
-# If true, using notify URL address with non-routable IPv4 and IPv6 addresses are allowed.
-# In case of non-routable addresses the IP packets cannot be directed from one network to another.
-allow_non_routable_addressing=true
+# If true, address detection from HttpServletRequest will be performed at the time of systems and service registration/unregistration
```

Comment:
comment is not talk about executors, also maybe the value should be true (?)

Reply:
false is the predifined in SR as well